src/api/OnmsHTTPOptions.ts

```diff
@@ -48,24 +48,30 @@ export class OnmsHTTPOptions {
   /** HTTP data to be passed when POSTing */
   public data: any;
 
+  /**
+   * The default timeout associated with these options.
+   *
+   * This is a trick for making sure serialization to JSON happens properly
+   * without exposing internals.
+   */
   private [TIMEOUT_PROP]: number;
 
+  /**
+   * The default authentication credentials associated with these options.
+   *
+   * This is a trick for making sure serialization to JSON happens properly
+   * without exposing internals.
+   */
   private [AUTH_PROP]: OnmsAuthConfig;
 
   /**
    * Construct a new OnmsHTTPOptions object.
    * @constructor
    */
   constructor(timeout?: number, auth?: OnmsAuthConfig, server?: OnmsServer) {
-    if (timeout !== undefined) {
-      this.timeout = timeout;
-    }
-    if (auth !== undefined) {
-      this.auth = auth;
-    }
-    if (server !== undefined) {
-      this.server = server;
-    }
+    this.timeout = timeout || DEFAULT_TIMEOUT;
```

Comment:
Give `if` noch chance 👍

Reply:
I'm not sure what your feedback is saying.  Are you requesting a change here?